functests/4_latency/latency.go

```diff
@@ -183,7 +183,10 @@ var _ = Describe("[performance] Latency Test", func() {
 		})
 
 		It("should succeed", func() {
```

Comment:
maybe we can elaborate more here : should succeed with values.. should succeed running latency tests with maximum X